// rmdi/lib/BubbleChart.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; };

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _styledComponents = require('styled-components');

var _styledComponents2 = _interopRequireDefault(_styledComponents);

var _styledSystem = require('styled-system');

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

function _objectWithoutProperties(obj, keys) { var target = {}; for (var i in obj) { if (keys.indexOf(i) >= 0) continue; if (!Object.prototype.hasOwnProperty.call(obj, i)) continue; target[i] = obj[i]; } return target; }

var Icon = function Icon(_ref) {
  var size = _ref.size,
      props = _objectWithoutProperties(_ref, ['size']);

  return _react2.default.createElement(
    'svg',
    _extends({}, props, {
      viewBox: '0 0 24 24',
      width: size,
      height: size,
      fill: 'currentcolor'
    }),
    _react2.default.createElement('path', { d: 'M 7.2 11.2 A 3.2 3.2 0 0 1 7.2 17.6 A 3.2 3.2 0 0 1 7.2 11.2 M 14.8 16 A 2 2 0 0 0 14.8 20 A 2 2 0 0 0 14.8 16 M 15.2 4.000000000000001 A 4.8 4.8 0 0 1 15.2 13.600000000000001 A 4.8 4.8 0 0 1 15.2 4.000000000000001' })
  );
};

var BubbleChartIcon = (0, _styledComponents2.default)(Icon)([], _styledSystem.space, _styledSystem.color);

BubbleChartIcon.displayName = 'BubbleChartIcon';

BubbleChartIcon.defaultProps = {
  size: 24
};

exports.default = BubbleChartIcon;
